.github/workflows/download_trivy_db.yml

```diff
@@ -20,19 +20,17 @@ jobs:
         run: |
           mkdir -p $GITHUB_WORKSPACE/.cache/trivy/db
           # try GHCR, fallback to ECR
-          { oras pull ghcr.io/aquasecurity/trivy-db:2 } || { oras pull public.ecr.aws/aquasecurity/trivy-db:2 }
+          { oras pull ghcr.io/aquasecurity/trivy-db:2; } || { oras pull public.ecr.aws/aquasecurity/trivy-db:2; }
```

Comment:
I believe we can drop the curly braces, ie: ```oras pull ghcr.io/aquasecurity/trivy-db:2 || oras pull public.ecr.aws/aquasecurity/trivy-db:2```

Reply:
You're right. I initially added them for clarity but given that they actually might have introduced a bug, it's probably better to lose them